### PR TITLE
adds operational issue log

### PR DESCRIPTION
**Pull request by CarlosCoelhoSL**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Adds an operational issue log to the pipeline run.
This log will track internal issues that the team have control over and will be tracked over time so the data management team can analyse their performance

## Related Tickets & Documents

- Ticket Link https://trello.com/c/J5GgSZR3/3484-create-operationalissuelog-in-digital-land-python

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] Are there any dependencies on other PRs or Work?

Requires a makerules PR to be merged first: https://github.com/digital-land/makerules/pull/42

### CODE REVIEW

File: digital_land/cli.py [hunk 1]
```diff
@@ -176,6 +178,8 @@ def dataset_dump_flattened_cmd(ctx, input_path, output_path):
 @dataset_resource_dir
 @organisation_path
 @collection_dir
+@performance_dir
```
Comment (eveleighoj) [inline digital_land/cli.py:181]: You probably just need the performance directory not both as the operational issue directory is in the performance directory.

File: digital_land/command_arguments.py [hunk 2]
```diff
@@ -35,6 +35,24 @@ def issue_dir(f):
...
+def performance_dir(f):
+    return click.option(
+        "--performance-dir",
+        "-i",
+        type=click.Path(exists=True),
+        default="performance/",
+    )(f)
+
+
+def operational_issue_dir(f):
```
Comment (eveleighoj) [inline digital_land/command_arguments.py:47]: we're moving this to the performance directory so this default may cause it to break

File: digital_land/commands.py [hunk 3]
```diff
@@ -163,6 +168,8 @@ def pipeline_run(
     collection_dir,  # TBD: remove, replaced by endpoints, organisations and entry_date
     null_path=None,  # TBD: remove this
     issue_dir=None,
+    performance_dir="performance/",
```
Comment (eveleighoj) [inline digital_land/commands.py:171]: again only need performance here could remove operational

File: digital_land/commands.py [hunk 4]
```diff
@@ -282,6 +293,9 @@ def pipeline_run(
     issue_log = duplicate_reference_check(issues=issue_log, csv_path=output_path)
 
     issue_log.save(os.path.join(issue_dir, resource + ".csv"))
+    operational_issue_log.save(
+        operational_issue_dir=operational_issue_dir, performance_dir=performance_dir
```
Comment (eveleighoj) [inline digital_land/commands.py:297]: maybe just call this output_dir in the save command command. then a directory can be passed in.

File: digital_land/command_arguments.py (hunk 2 above)
Comment (CarlosCoelhoSL) [reply digital_land/command_arguments.py:38]: I've updated this so the default is performance/operational_issue

**Review by eveleighoj — APPROVED** [review]

**Pull request merged**